packages/mobile/__mocks__/react-native-reanimated.js

```diff
@@ -0,0 +1 @@
+module.exports = require('react-native-reanimated/mock')
```

Comment:
Sounds good. Needing to add the mock in the linked PR makes more sense to me since it adds new `react-native-reanimated` imports, while this PR doesn't. I'm surprised tests weren't failing on master, but if this works, LGTM